d2l-user-card.js

```diff
@@ -119,13 +119,10 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-user-card">
 			}
 		</style>
 
-		<d2l-card aria-label$="[[name]]" loading$="[[_placeholders]]" href="javascript:void(0);">
+		<d2l-card text$="[[name]]" loading$="[[_placeholders]]" href="javascript:void(0);">
```

Comment:
```suggestion
		<d2l-card text="[[name]]" loading$="[[_placeholders]]" href="javascript:void(0);">
```

`$=` will set an attribute, rather than the property